README.md

```diff
@@ -63,7 +63,7 @@ sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10
 echo "deb http://repo.mongodb.org/apt/ubuntu "$(lsb_release -sc)"/mongodb-org/3.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-3.0.list
 sudo apt-get update
 sudo apt-get install -y mongodb-org
-sudo service mongod start
+sudo service mongodb start
```

Comment:
@obliviateandsurrender make sure only this change is in the PR. Please discard everything else.